rs/nns/integration_tests/src/node_provider_remuneration.rs

```diff
@@ -70,13 +70,23 @@ impl NodeInfo {
 
 #[test]
 fn test_list_node_provider_rewards() {
+    do_test_list_node_provider_rewards(&[]);
+}
+
+// TODO - after Node Reward Canister is by-default enabled, remove this test.
```

Comment:
```suggestion
// TODO[NNS1-XXXX] - after Node Reward Canister is by-default enabled, remove this test.
```

just to remember to address this